messages/TypescriptEslintStrategy.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-	"FileNotIncludedByTsConfig": "'%s' does not reside in a location that is included by your %s 'include' attribute.",
-	"MissingTsConfigFromCwd": "Unable to find '%s' in current directory '%s'. Please change to a directory that contains '%s' or specify it using the --tsconfig flag",
-	"InvalidNameTsConfigFromOptions": "File '%s' specified by the --tsconfig flag must be named '%s'",
-	"NotAFileTsConfigFromOptions": "Unable to find '%s' at location '%s' specified by the 'tsconfig' option"
+	"FileNotIncludedByTsConfig": "'%s' doesn't reside in a location that is included by your %s 'include' attribute.",
+	"MissingTsConfigFromCwd": "We couldn't find '%s' in the current directory '%s'. Switch to a directory that contains '%s' or specify the current directory with the --tsconfig flag.",
+	"InvalidNameTsConfigFromOptions": "The file '%s' specified by the --tsconfig flag must be named '%s'.",
+	"NotAFileTsConfigFromOptions": "We couldn't find the 'tsconfig.json' at location '%s' specified by the --tsconfig option."
```

Comment:
For consistency between line 4 and line 5, can we take 'tsconfig.json' file name from a parameter as well instead of hardcoding?